test/components/room_test.js

```diff
@@ -28,6 +28,37 @@ describe("Room component", () => {
         expect(roomComponent.find(StageProgressionButton)).to.have.length(1)
       })
     })
+
+    context("and there are no action items during the action-items stage", () => {
+      it("renders a disabled <StageProgressionButton>", () => {
+        const roomComponent = shallow(
+          <Room
+            {...defaultProps}
+            currentUser={facilitatorUser}
+            stage="action-items"
+          />
+        )
+        const stageProgressionButton = roomComponent.find(StageProgressionButton)
+        expect(stageProgressionButton).to.have.length(1)
+        expect(stageProgressionButton.prop("buttonDisabled")).to.be.true // eslint-disable-line
```

Comment:
@vanderhoop because of this: https://github.com/eslint/eslint/issues/2102

I'll add the name of the rule to be more clear.